### PR TITLE
Add PT-BR automatic translation maintenance_github_grooming.pt.Rmd

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: maintenance_github_grooming.pt.Rmd [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+aliases: grooming.html
+---
+
+# GitHub Grooming {#grooming}
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:5]:
```suggestion
# Preparação do GitHub {#grooming}
```

File: maintenance_github_grooming.pt.Rmd [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+```{block, type="summaryblock"}
+Atualmente, os pacotes do rOpenSci são, em sua grande maioria, desenvolvidos no GitHub. Aqui estão algumas dicas para aproveitar a plataforma em uma seção sobre [tornar seu repositório mais detectável] (#repodiscoverability) e uma seção sobre [comercializar sua própria conta do GitHub após passar pela revisão por pares] (#marketown).
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:8]:
```suggestion
Atualmente, os pacotes do rOpenSci são, em sua grande maioria, desenvolvidos no GitHub. Aqui estão algumas dicas para aproveitar a plataforma em uma seção sobre [tornar seu repositório mais detectável](#repodiscoverability) e uma seção sobre [comercializar sua própria conta do GitHub após passar pela revisão por pares](#marketown).
```

File: maintenance_github_grooming.pt.Rmd [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+```
+
+## Torne seu repositório mais detectável {#repodiscoverability}
+
+### Tópicos do repositório do GitHub {#git-hub-repo-topics}
+
+GitHub [tópicos do repositório](https://blog.github.com/2017-01-31-introducing-topics/) ajudam a navegar e pesquisar repositórios do GitHub, são usados por [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são digeridos por [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave do registro rOpenSci.
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:16]:
```suggestion
[Tópicos de repositórios](https://blog.github.com/2017-01-31-introducing-topics/) do GitHub ajudam a navegar e pesquisar repositórios do GitHub, são usados pelo [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são digeridos por [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave de registro da rOpenSci.
```

File: maintenance_github_grooming.pt.Rmd [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Recomendamos que você:
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:18]: "Recomendamos que você" followed by "adicionar" in the next lines doesn't add up.

```suggestion
Recomendamos:
```

File: maintenance_github_grooming.pt.Rmd [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacotes.
+
+- Adicionar quaisquer outros tópicos relevantes ao repositório de pacotes que você possui.
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:22]: I assume it is 1 repository per package. Or am I wrong and it is common in R environment to have more than one package per repo?

```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacote.

- Adicionar quaisquer outros tópicos relevantes ao repositório de pacote que você possui.
```

File: maintenance_github_grooming.pt.Rmd [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+Poderemos fazer sugestões a você depois que seu pacote for integrado.
+
+### Linguista do GitHub {#git-hub-linguist}
+
+[Linguista do GitHub](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".
+
+Recomendamos que você substitua o linguista do GitHub adicionando ou modificando um .gitattributes ao seu repositório em dois casos:
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:30]:
```suggestion
### GitHub linguist {#git-hub-linguist}

[GitHub linguist](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".

Recomendamos que você substitua o GitHub linguist adicionando ou modificando um .gitattributes ao seu repositório em dois casos:
```

File: maintenance_github_grooming.pt.Rmd [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+- Se você armazenar arquivos html em locais não padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicionar `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:32]:
```suggestion
- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```

File: maintenance_github_grooming.pt.Rmd [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+- Se o seu repositório contiver código que você não criou, por exemplo, código JavaScript, adicione `inst/js/* linguist-vendored` a .gitattributes ([Exemplo na natureza](https://github.com/ropensci/wellknown/blob/4435eb620eeae346d2cab7d62276c29dee29a898/.gitattributes#L1))
+
+Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguista não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).
+
+Mais informações sobre substituições de linguistas no GitHub [aqui](https://github.com/github/linguist#overrides).
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:38]:
```suggestion
Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguist não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).

Mais informações sobre substituições de GitHub linguist [aqui](https://github.com/github/linguist#overrides).
```

File: maintenance_github_grooming.pt.Rmd [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+## Comercialize sua própria conta {#marketown}
+
+- Como autor de um pacote integrado, você agora é membro da organização "ropensci" do rOpenSci no GitHub. Por padrão, as associações à organização são privadas; consulte [como torná-la pública nos documentos do GitHub](https://help.github.com/articles/publicizing-or-hiding-organization-membership/).
```
Comment (rffontenelle) [inline maintenance_github_grooming.pt.Rmd:42]:
```suggestion
- Como autor de um pacote integrado, você agora é membro da organização "ropensci" da rOpenSci no GitHub. Por padrão, as participações da organização são privadas; consulte [como torná-la pública na documentação do GitHub](https://help.github.com/articles/publicizing-or-hiding-organization-membership/).
```

**Review by rffontenelle — CHANGES_REQUESTED** [review]
Here is the first review

**Comment by beatrizmilz** [comment]
@arianacabral Disse que vai fazer a segunda revisão! 🎉

**Comment by arianacabral** [comment]
I'm about to begin the second review.

File: maintenance_github_grooming.pt.Rmd (hunk 3 above)
Comment (arianacabral) [reply maintenance_github_grooming.pt.Rmd:16]:
```suggestion
Os [tópicos de repositórios](https://blog.github.com/2017-01-31-introducing-topics/) do GitHub ajudam a navegar e pesquisar repositórios do GitHub, são usados pelo [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são processados pelo [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave de registro da rOpenSci.
```

File: maintenance_github_grooming.pt.Rmd [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+## Torne seu repositório mais detectável {#repodiscoverability}
+
+### Tópicos do repositório do GitHub {#git-hub-repo-topics}
+
+GitHub [tópicos do repositório](https://blog.github.com/2017-01-31-introducing-topics/) ajudam a navegar e pesquisar repositórios do GitHub, são usados por [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são digeridos por [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave do registro rOpenSci.
+
+Recomendamos que você:
+
+- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacotes.
```
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:20]:
```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao repositório de seu pacote.
```

File: maintenance_github_grooming.pt.Rmd (hunk 5 above)
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:22]:
```suggestion
- Adicionar quaisquer outros tópicos relevantes ao repositório do seu pacote.
```

File: maintenance_github_grooming.pt.Rmd [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+
+- Adicionar quaisquer outros tópicos relevantes ao repositório de pacotes que você possui.
+
+Poderemos fazer sugestões a você depois que seu pacote for integrado.
+
+### Linguista do GitHub {#git-hub-linguist}
+
+[Linguista do GitHub](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".
```
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:28]:
``` suggestion
O [GitHub linguist](https://github.com/github/linguist) atribuirá uma linguagem ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código em C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade de adicionar os tópicos "r", "r-package" e "rstats".
```

File: maintenance_github_grooming.pt.Rmd (hunk 7 above)
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:32]:
```suggestion
- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo em uso real](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```

File: maintenance_github_grooming.pt.Rmd [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+Recomendamos que você substitua o linguista do GitHub adicionando ou modificando um .gitattributes ao seu repositório em dois casos:
+
+- Se você armazenar arquivos html em locais não padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicionar `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
+
+- Se o seu repositório contiver código que você não criou, por exemplo, código JavaScript, adicione `inst/js/* linguist-vendored` a .gitattributes ([Exemplo na natureza](https://github.com/ropensci/wellknown/blob/4435eb620eeae346d2cab7d62276c29dee29a898/.gitattributes#L1))
```
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:34]:
```suggestion
- Se o seu repositório contiver código que você não criou, por exemplo, código JavaScript, adicione `inst/js/* linguist-vendored` a .gitattributes ([Exemplo em uso real](https://github.com/ropensci/wellknown/blob/4435eb620eeae346d2cab7d62276c29dee29a898/.gitattributes#L1))
```

File: maintenance_github_grooming.pt.Rmd (hunk 7 above)
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:32]: Acho que fica estranho deixar *Exemplo na natureza*. Eu sugeri *Exemplo em uso real*, mas ainda estou na dúvida se é a melhor opção.

File: maintenance_github_grooming.pt.Rmd [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+
+Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguista não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).
```
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:36]:
```suggestion
Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o GitHub linguist não reconhecer corretamente que seu repositório contém principalmente código R, seu pacote não aparecerá nos resultados de pesquisa usando o filtro `language:R`. Da mesma forma, seu repositório não poderá ser listado entre os [repositórios R em alta](https://github.com/trending/r).
```

File: maintenance_github_grooming.pt.Rmd (hunk 8 above)
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:38]:
```suggestion
Mais informações sobre as substituições do GitHub linguist podem ser encontradas [aqui](https://github.com/github/linguist#overrides).
```

File: maintenance_github_grooming.pt.Rmd [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+Mais informações sobre substituições de linguistas no GitHub [aqui](https://github.com/github/linguist#overrides).
+
+## Comercialize sua própria conta {#marketown}
+
+- Como autor de um pacote integrado, você agora é membro da organização "ropensci" do rOpenSci no GitHub. Por padrão, as associações à organização são privadas; consulte [como torná-la pública nos documentos do GitHub](https://help.github.com/articles/publicizing-or-hiding-organization-membership/).
+
+- Mesmo depois que o repositório do seu pacote tiver sido transferido para o rOpenSci, você pode [fixá-lo em sua própria conta](https://help.github.com/articles/pinning-repositories-to-your-profile/).
```
Comment (arianacabral) [inline maintenance_github_grooming.pt.Rmd:44]:
```suggestion
- Mesmo após o repositório do seu pacote ser transferido para a rOpenSci, você pode [fixá-lo em sua conta pessoal](https://help.github.com/articles/pinning-repositories-to-your-profile/).
```

File: maintenance_github_grooming.pt.Rmd (hunk 5 above)
Comment (yabellini) [reply maintenance_github_grooming.pt.Rmd:22]: It is correct, it is one repo per package.

**Comment by yabellini** [comment]
@rffontenelle, I commit your reviews and leave @arianacabral suggestions for when you have the time to give an okay or continue to comment on Ariana's suggestions. If you react with a thumbs-up, it is enough for me to know I can commit.  Thnk you both for this work!

**Comment by rffontenelle** [comment]
@arianacabral Thanks for these hawk Eyes on finding all these points of improvements!

@yabellini all good to go! Thanks!

File: maintenance_github_grooming.pt.Rmd [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+## Torne seu repositório mais detectável {#repodiscoverability}
+
+### Tópicos do repositório do GitHub {#git-hub-repo-topics}
+
+Os [tópicos de repositórios](https://blog.github.com/2017-01-31-introducing-topics/) do GitHub ajudam a navegar e pesquisar repositórios do GitHub, são usados pelo [R-universe em páginas de pacotes e para resultados de pesquisa](https://github.com/r-universe-org/help#how-to-add-keyword-labels-to-an-r-package) e são processados pelo [`codemetar`](https://github.com/ropensci/codemetar) para palavras-chave de registro da rOpenSci.
+
+Recomendamos:
+
+- Adicionar "r", "r-package" e "rstats" como tópicos ao seu repositório de pacote.
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:20]:
```suggestion
- Adicionar "r", "r-package" e "rstats" como tópicos ao repositório de seu pacote.
```

File: maintenance_github_grooming.pt.Rmd [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+
+- Adicionar quaisquer outros tópicos relevantes ao repositório de pacote que você possui.
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:22]:
```suggestion
- Adicionar quaisquer outros tópicos relevantes ao repositório do seu pacote.
```

File: maintenance_github_grooming.pt.Rmd [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+Poderemos fazer sugestões a você depois que seu pacote for integrado.
+
+### GitHub linguist {#git-hub-linguist}
+
+[GitHub linguist](https://github.com/github/linguist) atribuirá um idioma ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade dos tópicos "r", "r-package" e "rstats".
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:28]:
```suggestion
O [GitHub linguist](https://github.com/github/linguist) atribuirá uma linguagem ao seu repositório com base nos arquivos que ele contém. Alguns pacotes que contêm muito código em C++ podem ser classificados como pacotes C++ em vez de pacotes R, o que é bom e mostra a necessidade de adicionar os tópicos "r", "r-package" e "rstats".
```

File: maintenance_github_grooming.pt.Rmd [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+
+Recomendamos que você substitua o GitHub linguist adicionando ou modificando um .gitattributes ao seu repositório em dois casos:
+
+- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo na natureza](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:32]:
```suggestion
- Se você armazenar arquivos html em locais diferentes do padrão (não em docs/, por exemplo, em vignettes/), use as substituições de documentação. Adicione `*.html linguist-documentation=true` ao arquivo .gitattributes ([Exemplo em uso real](https://github.com/ropensci/ecoengine/blob/56b64d8d29dfae430a776d1dd440b240452eb1bf/.gitattributes#L5))
```

File: maintenance_github_grooming.pt.Rmd [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+
+- Se o seu repositório contiver código que você não criou, por exemplo, código JavaScript, adicione `inst/js/* linguist-vendored` a .gitattributes ([Exemplo em uso real](https://github.com/ropensci/wellknown/blob/4435eb620eeae346d2cab7d62276c29dee29a898/.gitattributes#L1))
+
+Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o linguist não reconhecer corretamente o seu repositório como contendo principalmente código R, seu pacote não aparecerá nos resultados de pesquisa com a opção `language:R` você não aparecerá nos resultados da pesquisa. Da mesma forma, seu repositório não poderá ser listado entre os pacotes [repositórios R em alta](https://github.com/trending/r).
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:36]:
```suggestion
Dessa forma, a classificação da linguagem e as estatísticas do seu repositório refletirão melhor o código-fonte que ele contém, além de torná-lo mais detectável. Notavelmente, se o GitHub linguist não reconhecer corretamente que seu repositório contém principalmente código R, seu pacote não aparecerá nos resultados de pesquisa usando o filtro `language:R`. Da mesma forma, seu repositório não poderá ser listado entre os [repositórios R em alta](https://github.com/trending/r).
```

File: maintenance_github_grooming.pt.Rmd [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+
+Mais informações sobre substituições de GitHub linguist [aqui](https://github.com/github/linguist#overrides).
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:38]:
```suggestion
Mais informações sobre as substituições do GitHub linguist podem ser encontradas [aqui](https://github.com/github/linguist#overrides).
```

**Review by yabellini — COMMENTED** [review]
All these suggestions were made by @arianacabral in the second review of the translation in this PR.

File: maintenance_github_grooming.pt.Rmd [hunk 21]
```diff
@@ -0,0 +1,48 @@
+---
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:1]:
```suggestion

```

File: maintenance_github_grooming.pt.Rmd [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+aliases: grooming.html
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:2]:
```suggestion

```

File: maintenance_github_grooming.pt.Rmd [hunk 23]
```diff
@@ -0,0 +1,48 @@
...
+---
```
Comment (yabellini) [inline maintenance_github_grooming.pt.Rmd:3]:
```suggestion

```

**Review by yabellini — COMMENTED** [review]
removing YAML

**Pull request merged**